app/views/ketchups/_form.html.erb

```diff
@@ -29,9 +29,9 @@
 
   <div class="form-group">
     <%= form.label "Where do you want to Ketchup" %>
-    <%= form.text_field :location, class: 'form-control', data: { target: "places.field", action: "keydown->places#preventSubmit" } %>
-    <%= form.text_field :latitude, type: "hidden", data: { target: "places.latitude" }, class: 'form-control' %>
-    <%= form.text_field :longitude, type: "hidden", data: { target: "places.longitude" }, class: 'form-control' %>
+    <%= form.text_field :location, class: 'form-control',  onkeypress: "return event.keyCode!=13" %>
```

Comment:
what is this line doing?

Reply:
Preventing submit on pressing enter key. It's working for now but will try to find a way to make a function later.